packages/build/tests/functions/snapshots/tests.js.md

```diff
@@ -334,11 +334,15 @@ Generated by [AVA](https://avajs.dev).
     ␊
     ␊
     ────────────────────────────────────────────────────────────────␊
-      Configuration error                                           ␊
+      Internal error during "Functions bundling"                    ␊
```

Comment:
Why do we have a different result here?

Reply:
I don't know, i guess nft and zisi work differently? The underlying issue seems to be the same.